docs/MAINTAINING.md

```diff
@@ -0,0 +1,43 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Maintaining
+
+- [Maintaining](#maintaining)
+  - [Workflow](#workflow)
+    - [Milestones](#milestones)
+    - [Project Boards](#project-boards)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Maintaining
+
+Current maintainers:
+
+- @code-asher
+- @oxy
+- @jsjoeio
+
+This document is meant to serve current and future maintainers of code-server, but also share openly our workflow for maintaining the project.
+
+## Workflow
+
+The workflow used by code-server maintainers is one that aims to be easy to understood by the community and easy enough for new maintainers to jump in and start contributing on day one.
+
+### Milestones
+
+We operate mainly using [Milestones](https://github.com/cdr/code-server/milestones). This was heavily inspired by our friends over at [vscode](https://github.com/microsoft/vscode).
+
+Here are the Milestones we use and how we use them:
+
+- "Backlog" -> Work not yet planned for a specific release.
```

Comment:
So with this any un-assigned issues are essentially in triage state and once triaged we add to either "backlog" or "backlog candidates", right? Then eventually they move to "on deck" and then at some point to a milestone where they will finally be worked on.

Reply:
Exactly! I think you explained it perfectly. I'll add this too